libs/lizard-connector/lizard_connector/queries.py

```diff
@@ -0,0 +1,204 @@
+            *queries (iterable): iterable containing dicts to update with.
+            **f (dict): key, value pairs to update with.
+        """
+        for k, v in f.items():
+            self[k] = v
+        queries = list(queries)
+        if E and isinstance(E, str):
+            for k, v in urlparse.parse_qsl(E.strip('?')):
+                self[k] = v
+        elif E:
+            queries.append(E)
+        for d in queries:
+            try:
+                for k, v in d.items():
+                    self[k] = v
+            except AttributeError:
+                raise LizardApiImproperQueryError(
+                    'Query {} is not a dictionary or a string'.format(E))
+
+
+def commaify(*args):
+    """
+    Returns a comma-seperated string of the given arguments (str).
+    """
+    return ','.join(str(x) for x in args)
+
+
+def wkt_point(lon, lat):
+    return ''.join(('POINT (', str(lon), ' ', str(lat), ')'))
+
+
+def wkt_polygon(polygon_coordinates):
+    points = [' '.join([str(x), str(y)]) for x, y in polygon_coordinates]
+    return 'POLYGON ((' + ', '.join(points) + '))'
+
+
+def bbox(south_west, north_east):
+    """
+    Creates a bounding box in Well-known text (WKT).
+
+    Args:
+        south_west (list, tuple): coordinates of the most south western
+                                  point of the bounding box of the form
+                                  (lat, lon).
+        north_east (list, tuple): coordinates of the most north eastern
+                                  point of the bounding box of the form
+                                  (lat, lon).
+
+    Returns:
+        bounding box polygon in Well-known text (WKT).
+    """
+    min_lat, min_lon = south_west
+    max_lat, max_lon = north_east
+    polygon_coordinates = [
+        [min_lon, min_lat],
+        [min_lon, max_lat],
+        [max_lon, max_lat],
+        [max_lon, min_lat],
+        [min_lon, min_lat],
+    ]
+    return wkt_polygon(polygon_coordinates)
+
+
+def in_bbox(south_west, north_east, endpoint=None):
+    """
+    Find all locations within a certain bounding box.
+
+    returns records within bounding box using Bounding Box format (min Lon,
+    min Lat, max Lon, max Lat). Also returns features with overlapping
+    geometry.
+
+    Args:
+        endpoint (str): endpoint name as it is available in the Lizard-api
+        south_west (list, tuple): coordinates of the most south western
+                                  point of the bounding box of the form
+                                  (lat, lon).
+        north_east (list, tuple): coordinates of the most north eastern
+                                  point of the bounding box of the form
+                                  (lat, lon).
+
+    Returns:
+        a query dictionary.
+    """
+    query = "geom_within" if endpoint == 'timeseries' else "in_bbox"
+    return QueryDictionary({query: bbox(south_west, north_east)})
+
+
+def distance_to_point(distance, lat, lon):
```

Comment:
I get why you use ``lat/lon`` (ISO standard for geographic point locations). But you should be aware that it is often assumed that this is just a different way of saying ``x/y``, which it is not.